docs/getting-started.md

```diff
@@ -171,14 +184,17 @@ Now that you're setup, if you can run it on the command line, you can run it in
 5. Run Robot Diagnostics: `bazel run //software/gui/robot_diagnostics:robot_diagnostics_main -- --interface=[interface_here] --backend=WifiBackend`
     - The Mechanical and Electrical sub-teams use Robot Diagnostics to test specific parts of the Robot.
 
+### Running AI vs AI
+1. Open your terminal, `cd` into `Software/src`
+2. Run `./software/run_ai_vs_ai.sh interface_name`, using the same interface as from [above](#running-our-ai-simulator-or-robot-diagnostics)
 
 ## Debugging
 Debugging from the command line is certainly possible, but debugging in a full IDE is *really* nice (plz trust us). 
 
-### With CLion
+### Debugging with CLion
 Debugging in CLion is as simple as running the above instructions for building CLion, but clicking the little green bug in the top right corner instead of the little green arrow!
 
-### From The Command line
+### Debugging from the Command line
```

Comment:
Small nit, we refer to it as `command-line` earlier. Non-blocking but wanted to point it out.

Reply:
Made https://github.com/UBC-Thunderbots/Software/issues/1854 to track it.